Userland/Libraries/LibGfx/ImageFormats/ILBMLoader.cpp

```diff
@@ -278,14 +266,13 @@ static ErrorOr<void> decode_iff_chunks(ILBMLoadingContext& context)
         auto chunk = TRY(decode_iff_advance_chunk(chunks));
         if (chunk.type == FourCC("CMAP"))
             context.color_table = TRY(decode_cmap_chunk(chunk));
-        else if (chunk.type == FourCC("BODY"))
+        else if (chunk.type == FourCC("BODY")) {
             TRY(decode_body_chunk(chunk, context));
-        else if (chunk.type == FourCC("CRNG"))
+            context.state = ILBMLoadingContext::State::BitmapDecoded;
+        } else if (chunk.type == FourCC("CRNG"))
```

Comment:
https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md#omission-of-curly-braces-from-statement-blocks

You're not respecting this in `decode_body_chunk`